Updated scripts from Hololens 2 work

dstrawberrygirl opened the pull request:
This is all the script updates from the Hololens 2 work. A lot of small fixes and duck tape to make sure everything is correctly referenced and positioned correctly. 

This does produce a build that has the same issue as the Hololens version in that annotations are not currently working, this will be my focus tomorrow morning and I'll update this PR when I fix it.

knowuh reviewed (approved):
Looks good. I think I had a few questions where comments might have helped me out.

dstrawberrygirl commented:
Thanks @knowuh for the review! I have added comments where appropriate. 
I also added a very hasty tutorial screen (it's a picture of controllers & mouse/keyboard for hits on how to use the application). I had this reviewed in part by Nathan, will have QA test it out in the field and if we need to tweak I'll add another PR :)
